test/book/specialized/decker-schmitt-invariant-theory/H3.jlcon

```diff
@@ -60,9 +60,9 @@ julia> L = syzygy_generators([hessian(G), F, hessian(F)]);
 
 julia> collect(coefficients(L[1]))
 3-element Vector{AbstractAlgebra.Generic.FracFieldElem{QQPolyRingElem}}:
- -1
- -279936*t
  t^3 + 93312
+ -279936*t
+ -1
```

Comment:
In `master` we get
```
julia> L[1]
-e[1] - 279936*t*e[2] + (t^3 + 93312)*e[3]

julia> coefficients(L[1])
coefficients iterator of -e[1] - 279936*t*e[2] + (t^3 + 93312)*e[3]

julia> collect(coefficients(L[1]))
3-element Vector{AbstractAlgebra.Generic.FracFieldElem{QQPolyRingElem}}:
 -1
 -279936*t
 t^3 + 93312
```
In this branch we get
```
julia> L[1]
-e[1] - 279936*t*e[2] + (t^3 + 93312)*e[3]

julia> coefficients(L[1])
coefficients iterator of (t^3 + 93312)*e[3] - 279936*t*e[2] - e[1]

julia> collect(coefficients(L[1]))
3-element Vector{AbstractAlgebra.Generic.FracFieldElem{QQPolyRingElem}}:
 t^3 + 93312
 -279936*t
 -1
```

So the iteration order changed but the result itself is fine.

Reply:
@fingolfin : @wdecker should have a look at this to make sure that the changed iteration order does not break any wording that might be refering to a 'first entry' or something similar. It is not about the computation result!.

Reply:
I checked and there does not seem to be such a problem. Also in the book there will anyway be the old output, won't it?